module-types/HPE/Aruba-JL085A.yaml

```diff
@@ -0,0 +1,9 @@
+---
+manufacturer: HPE
+model: JL085A
+part_number: JL085A
+comments: Aruba X371 250W AC Power Supply
+power-ports:
+  - name: Power {module}
```

Comment:
In the config, does this device actually call it `Power 1`, `Power 2`, etc? If not, please adjust this to the name matches what it does in the device config/output

Reply:
You are right, the output is as followed:
I will adjust to `PS {module}` 🙂 Thank you!

 PS#   Model     Serial      State           AC/DC  + V        Wattage   Max
 ----- --------- ----------- --------------- ----------------- --------- ------
 1     JL085A    CN04******  Powered         AC 120V/240V        44       250
 2                           Not Present     -- ---------         0         0